docs/troubleshooting.md

```diff
@@ -139,3 +139,7 @@ It is possible customer is seeing the error "transport endpoint is not connected
 - Try restarting/rebooting the VM Instance.
 
 If it's running on GKE, the issue could be caused by an Out-of-Memory (OOM) error. Consider increasing the memory allocated to the GKE sidecar container. For more info refer [here](https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/blob/main/docs/known-issues.md#implications-of-the-sidecar-container-design).
+
+### Not able to see some files and directories in mounted directory
+
+- If some files and directories that exist in your cloud storage bucket are not visible in the mounted directory, ensure that you included the `--implicit-dir` flag when mounting the bucket.
```

Comment:
I think we should link https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#files-and-directories section here so the customer is aware of the drawbacks and options that they have.